mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc

```diff
@@ -581,16 +574,23 @@ class ModelAPISideFeedCalculator : public CalculatorBase {
                 LOG(INFO) << "OVMS calculator will process vector<TfLiteTensor>";
                 auto outputStreamTensors = std::vector<TfLiteTensor>();
                 if (!this->initialized) {
-                    interpreter_->AddTensors(output.size()); // HARDCODE
-                    interpreter_->SetInputs({0,1}); // HARDCODE was 0 for single input
+                    interpreter_->AddTensors(output.size());
+                    std::vector<int> indexes(output.size());
+                    std::iota(indexes.begin(), indexes.end(), 0);
+                    interpreter_->SetInputs(indexes);
+                    RET_CHECK(output.size() == 2);
                     size_t tensorId = 0;
                     for (auto& [name,tensor] : output) {
                         std::vector<int> tfliteshape;
                         for (auto& d : tensor.get_shape()) {
                             tfliteshape.emplace_back(d);
                         }
-                        interpreter_->SetTensorParametersReadWrite(/*tensor_index*/tensorId, kTfLiteFloat32, name.c_str(),
-                                                       tfliteshape, TfLiteQuantization());
+                        interpreter_->SetTensorParametersReadWrite(
+                                        /*tensor_index*/tensorId,
+                                        kTfLiteFloat32, // TODO datatype
```

Comment:
Do we support other precisions than float32 in this release?

Reply:
Not for TfLite